docroot/modules/humsci/hs_layouts/patterns/spotlight/spotlight.html.twig

```diff
@@ -1,12 +1,15 @@
-{%- set attributes = attributes.addClass('hb-spotlight') -%}
+{%- set attributes = attributes.addClass('hb-spotlight', bg_class) -%}
 
 {% if variant %}
   {% set variant_class = "hb-spotlight--image-#{variant}" %}
   {% set attributes = attributes.addClass(variant_class) %}
 {% endif %}
 
+{% set bg_class = background.field_hs_spotlight_bg['0']['#markup']|clean_class %}
```

Comment:
I don't like how this is handled. This could be better done in a preprocess hook perhaps without the need for a new region in the pattern. I believe the hook would be `hs_layouts_preprocess_pattern_spotlight`